package.json

```diff
@@ -14,12 +14,15 @@
     "build": "npm run lint && tsc",
     "generate": "npm run build && echo 'Processing, please wait.' && node built/."
   },
+  "bin": {
+    "test-data-generator": "src/built/index.js"
```

Comment:
Hi @jkuester, if you have a few minutes to spare, is it working for you when installing it globally? 

```
npm install -g
npm link
```

Reply:
Sorry for the delayed response here! Couple of observations:

- I needed to change this path to be `built/index.js` (without `src`)
- Also, I had to run `npm run build` so that the js files exist (maybe could be a `postinstall` script?)
- Not exactly sure what `npm link` is for (unless it is just for testing local changes to the repo via the global `test-data-generator` command...)

Reply:
> Also, I had to run npm run build so that the js files exist (maybe could be a postinstall script?)

I haven't found a way to do this while at the same time installing the tool globally. I get an error: 
<img width="964" alt="Screenshot 2024-06-17 at 6 33 51 PM" src="https://github.com/latin-panda/test-data-generator/assets/66472237/f6755f05-a64b-49d8-b87c-4b6eb23f369c">

From my research, it seems that this is typically done in `prepare` or `prepublish-only` before publishing in the npm registry. 

At the moment, the steps are:
- npm ci 
- npm build 
- npm install -g 

The `npm ci` and `npm build` can be merged into one, but still kinda weird initial setup haha

BTW, I feel that `test-data-generator [file path]` it's kinda long to type. Some other alternatives are:
- cht-tdg
- cht-test-data
- tdg
-____?

Reply:
> I haven't found a way to do this while at the same time installing the tool globally

Rats, that is annoying! (But I guess makes sense if you are only publishing the JS to npm...)

> it's kinda long to type

IMHO, `tdg` is great!

Reply:
Deal!, I updated to `tdg`